components/support/utils/src/main/java/mozilla/components/support/utils/URLStringUtils.kt

```diff
@@ -61,7 +61,7 @@ object URLStringUtils {
         // Be lenient about what is classified as potentially a URL.
         // (\w+-)*\w+(://[/]*|:|\.)(\w+-)*\w+([\S&&[^\w-]]\S*)?
```

Comment:
@codrut-topliceanu The regex in this comment still needs to be updated to `(\w+-+)` for the two groups otherwise it's confusing to read the new explanation :).

Reply:
Darn, you're right. Good eye! Sorry I missed it, fixing it now.

Reply:
👍